src/core/software/pupil-capture.md

```diff
@@ -500,3 +505,26 @@ Pupil Capture selects the active camera intrinsics following these priorities:
 1. Fallback to a "dummy calibration" ([pinhole camera model without distortion, focal length 1000px](https://github.com/pupil-labs/pupil/blob/master/pupil_src/shared_modules/camera_models.py#L659-L664)).
 
 Pupil Player follows the same priorities as Pupil Capture but expects the custom intrinsics to be present within the recording under the `<video file name>.intrinsics` file name pattern, e.g. `world.intrinsics`.
+
+#### Camera field of view (FOV)
+
+Based the estimated intrinsics, one can calculate the camera's field of view (FOV).
+
+**Field of view in degrees:**
+| Camera name                          | Resolution  | Horizontal | Vertical | Diagonal |
+| :---                                 | :---:       | :---:      | :---:    | :---:    |
+| Pupil Cam1 ID2 (wide-angle lens)     | `1920x1080` | 155°       | 85°      | ---      |
```

Comment:
```suggestion
| Pupil Cam1 ID2 (default - wide-angle lens)     | `1920x1080` | 155°       | 85°      | ---      |
```

Reply:
@papr One last change that I think will help users. We always use the wide angle lens for Pupil Core when it ships, so maybe a note that it is "default" would make sense here. Thoughts?